### ISSUE TITLE
5kov - problem with H chain

### ISSUE THREAD

**Issue by AndrewCRMartin** [AndrewCRMartin/absplit#28]
5kov - problem with H chain - weird chain names
```
       0P - ok but chain duplicated
       1P - ok but chain duplicated
       2P - not numbered (.bad) - confused by chain labels
       3P - ok but chain duplicated
       4P - ok? perhaps missing partner chain
       5P - OK
       6  - ok? perhaps missing partner chain
       7P - ok but chain duplicated
```

**Comment by AndrewCRMartin** [comment]
```
       0P - Fixed in fix for #29
       1P - Fixed in fix for #29
       2P - not numbered (.bad) - confused by chain labels
       3P - Fixed in fix for #29
       4P - ok? perhaps missing partner chain
       5P - OK
       6  - ok? perhaps missing partner chain
       7P - Fixed in fix for #29
```

**Comment by AndrewCRMartin** [comment]
Chain labelling now sorted - previously antigens with name of `H` or `L` were just downcased which messed things up with later stages of numbering. Now they are given lower case labels from `a` (and checks are made that these names are not already in use).

**Comment by AndrewCRMartin** [comment]
All fixed now:

0P - Fixed in fix for #29
1P - Fixed in fix for #29
2P - Fixed above
3P - #29
4P - Confirmed correct
5P - OK
6 - Confirmed correct
7P - Fixed in fix for #29
